.github/workflows/android.yml

```
@@ -13,10 +13,10 @@ jobs:
 
         steps:
             -   uses: actions/checkout@v6
-            -   name: set up JDK 17
+            -   name: set up JDK 21
```

YAMLlint (1.38.0)

[error] too many spaces after hyphen

(hyphens)